src/backend/tests/expunger/test_time_analyzer.py

```diff
@@ -81,40 +81,6 @@ def test_less_than_three_year_rule_conviction(self):
         assert charge.expungement_result.time_eligibility.reason == 'Most recent conviction is less than three years old'
         assert charge.expungement_result.time_eligibility.date_will_be_eligible == date.today() + relativedelta(days=+1)
 
-    def test_felony_class_b_greater_than_20yrs(self):
-        charge = ChargeFactory.create(name='Aggravated theft in the first degree',
-                                      statute='164.057',
-                                      level='Felony Class B',
-                                      date=Time.TWENTY_YEARS_AGO,
-                                      disposition=['Convicted', Time.TWENTY_YEARS_AGO])
-
-        self.expunger.class_b_felonies = [charge]
-        self.expunger.most_recent_charge = charge
-
-        self.expunger.charges = [charge]
-        TimeAnalyzer.evaluate(self.expunger)
-
-        assert charge.expungement_result.time_eligibility.status is EligibilityStatus.ELIGIBLE
-        assert charge.expungement_result.time_eligibility.reason == ''
-        assert charge.expungement_result.time_eligibility.date_will_be_eligible is None
-
-    def test_felony_class_b_less_than_20yrs(self):
-        charge = ChargeFactory.create(name='Aggravated theft in the first degree',
-                                      statute='164.057',
-                                      level='Felony Class B',
-                                      date=Time.LESS_THAN_TWENTY_YEARS_AGO,
-                                      disposition=['Convicted', Time.LESS_THAN_TWENTY_YEARS_AGO])
-
-        self.expunger.class_b_felonies = [charge]
-        self.expunger.most_recent_charge = charge
-
-        self.expunger.charges = [charge]
-        TimeAnalyzer.evaluate(self.expunger)
-
-        assert charge.expungement_result.time_eligibility.status is EligibilityStatus.INELIGIBLE
-        assert charge.expungement_result.time_eligibility.reason == 'Time-ineligible under 137.225(5)(a)(A)(i)'
-        assert charge.expungement_result.time_eligibility.date_will_be_eligible == Time.TOMORROW
-
```

Comment:
Creating a new file to put these tests in doesn't fit with the current style of tests.
We're testing the time_analyzer, hence the tests should stay in the time_analyzer test file to conform to the style of the project layout.<- if this style is not followed then we could potentially end up with a test file for each method.

If the desire is to separate these tests into their own section then I would suggest creating a class. Examples of this can be found in this file.

Reply:
Sure.